### DISCUSSION TITLE
Issue with physical dimension of converted png image

### DISCUSSION THREAD

**Discussion by wildecats** [ImageMagick/ImageMagick#7020] [Help] [answered]
### ImageMagick version

7.1.1-26 Q16-HDRI x86_64 21914

### Operating system, version and so on

macOS Monterey version 12.4

### Description

I am using the following command to convert a pdf file into a png image file: "convert -density 200 input.pdf -quality 100 -resize 100% output.png". When I open the converted png image to view it in the Preview app on my mac, the image looks fine except that it is unexpectedly positioned in the top right corner of the window. The image width and height take up only about 70% of the total window width and height, respectively. When I open the input pdf file, the image that appears on the screen takes up 100% of the window width and height. I would like for the output png file to also take up 100% of the window width and height (so that when I import the image to powerpoint, it loads nicely), but I'm not sure what options I need to use. Also, the output png image has an "image size" of 2339 x 1653 pixels when I click on the Information icon in the Preview app. Attached are my input.pdf and output.png files. I apologize for this very elementary question. I don't have much experience using imagemagick my web searches did not result in much help.

Here is my output.png file:
![output](https://github.com/ImageMagick/ImageMagick/assets/134729672/a88cd4df-d8d6-4448-aefa-24548f8df04c)

And here is my input.pdf file:
[input.pdf](https://github.com/ImageMagick/ImageMagick/files/13903800/input.pdf)

**Comment by snibgo** [comment, marked as answer]
As you have IM v7, I suggest you use `magick`, not `convert` or `magick convert`.

Your `-resize 100%` does nothing.

By default, IM ignores the cropbox in the PDF. You can use the cropbox like this (see https://imagemagick.org/script/defines.php):
```
magick -density 200 -define pdf:use-cropbox=true input.pdf out.png
```

> Also, the output png image has an "image size" of 2339 x 1653 pixels when I click on the Information icon in the Preview app.

What is the problem with that? If you want different dimensions, you can use a different `-density`, or `-resize` the image.

**Reply by wildecats** [reply]
Thanks for your help! This works exactly as I want and the image in the output .png file takes up the full window area. By the way, my comment about the image size being 2339 x 1653 pixels was just additional (irrelevant) information; it was not a problem that I needed a fix for.
